src/components/common/ColonyHome/ColonyHomeLayout/ColonyHomeLayout.tsx

```diff
@@ -1,84 +1,76 @@
 import React, { ReactNode } from 'react';
+import { useParams } from 'react-router-dom';
 
-import NewActionButton from '~common/NewActionButton';
 import ColonyTotalFunds from '~common/ColonyTotalFunds';
 import { useColonyContext } from '~hooks';
 
-import ColonyDomainSelector from './ColonyDomainSelector';
-import ColonyFundingWidget from './ColonyFundingWidget';
+import ColonyFundingWidget from '../ColonyFundingWidget';
 // import ColonyUnclaimedTransfers from './ColonyUnclaimedTransfers';
-import ColonyTitle from './ColonyTitle';
-import ColonyNavigation from './ColonyNavigation';
-import ColonyMembersWidget from './ColonyMembersWidget';
-// import ColonyExtensions from './ColonyExtensions';
-import ColonyDomainDescription from './ColonyDomainDescription';
-// import ColonyUpgrade from './ColonyUpgrade';
-// import ExtensionUpgrade from './ExtensionUpgrade';
+import ColonyTitle from '../ColonyTitle';
+import ColonyNavigation from '../ColonyNavigation';
+import ColonyMembersWidget from '../ColonyMembersWidget';
+// import ColonyExtensions from '../ColonyExtensions';
+import ColonyDomainDescription from '../ColonyDomainDescription';
+// import ColonyUpgrade from '../ColonyUpgrade';
+// import ExtensionUpgrade from '../ExtensionUpgrade';
+
+import ActionsPanel from './ActionsPanel';
 
 import styles from './ColonyHomeLayout.css';
 
-type Props = {
+export type Params = ReturnType<typeof useParams>;
```

Comment:
These params don't seem to be specific to ColonyHomeLayout, so why export them from here? Also could you not use the existing `Params` from `react-router-dom`?

Reply:
As far as I can tell, they don't export clean typings for `useParams`.

![paramsType](https://user-images.githubusercontent.com/64402732/214273348-e3bd9b40-7fde-48b7-ba48-0f0aaf938261.png)